source/backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -104,19 +104,19 @@ private static ExpressionStarter<PimsProperty> GenerateCommonPropertyQuery(Claim
                 ownershipBuilder = isRetired ? PredicateBuilder.New<PimsProperty>(p => p.IsRetired == true) : PredicateBuilder.New<PimsProperty>(p => false);
                 if (filter.Ownership.Contains("isCoreInventory"))
                 {
-                    ownershipBuilder = ownershipBuilder.Or(p => p.IsOwned);
+                    ownershipBuilder = ownershipBuilder.Or(p => p.IsOwned && p.IsRetired != true);
```

Comment:
!p.isRetired

Reply:
not quite, isRetired can be null or false.